docker-bake.hcl

```diff
@@ -14,6 +14,10 @@ variable "BUILDKITD_TAGS" {
   default = null
 }
 
+variable "BUILD_VARIANT" {
```

Comment:
As "variant" can mean pretty much anything, it might be better to call this debug. Or it can also be a separate target with "inherits".

Reply:
Just reverting this one. I didn't realize that the bake file didn't have a definition for building an image because I have a custom override that has it build an image so I didn't realize that wasn't a native portion of it. This variable doesn't really make sense for any of the bake targets.